nodeup/pkg/bootstrap/install.go

```diff
@@ -175,7 +175,7 @@ func (i *Installation) buildSystemdJob() *nodetasks.InstallService {
 	service := &nodetasks.InstallService{Service: nodetasks.Service{
 		Name:       serviceName,
 		Definition: fi.PtrTo(manifestString),
-		Enabled:    fi.PtrTo(false),
+		Enabled:    fi.PtrTo(true),
```

Comment:
like said in slack: if we just apply this change the service will be always enabled in systemd level. That another model kops-configuration.go does not matter anything.

Reply:
I am thinking could we enable this service BEFORE we start running normal kops-configuration process? In that case if systemd will restart the process it will start again? However, finally we could disable the service? Does it make sense? No idea does it help really

Reply:
You are right, this is not the right way to go about this. @justinsb suggested to return some specific message when node already registered and exit gracefully.

Reply:
that is actually good idea, then we could keep service enabled like before

Reply:
Exactly! :)

Reply:
Let's start with the reverts and we can do ^^^ later next week.
https://github.com/kubernetes/kops/pull/15129
https://github.com/kubernetes/kops/pull/15130